src/api.rs

```diff
@@ -10,6 +10,20 @@ use crate::helpers;
 use crate::responses::*;
 use storage_proofs::sector::SectorId;
 
+#[cfg(not(target_os = "windows"))]
+unsafe fn raw_to_file(raw: *mut libc::c_void) -> std::fs::File {
+    use std::os::unix::io::{FromRawFd, RawFd};
+
+    std::fs::File::from_raw_fd(raw as RawFd)
+}
+
+#[cfg(target_os = "windows")]
```

Comment:
I suggest that you drop this code block (unless you've actually tested that this works as-intended).

Reply:
I was going to test it on my windows machine, yes